### PR TITLE
Fix import errors of test suite

### PR DESCRIPTION
**Pull request by rim99**

### CODE REVIEW

**Comment by rim99** [comment]
Ignore the 2nd commit please. 
It may not be appropriate.
.

**Comment by rim99** [comment]
The 1st commit CAN make `testFavicon` in `test_core.py` pass.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Branch: "cheroot-py3"
```
Comment (webknjaz) [inline README.md]: Please revert the recent commit

**Comment by webknjaz** [comment]
> which cannot be deleted 

Why?

**Pull request closed without merging**